src/lib/logMsg/logMsg.h

```diff
@@ -1793,13 +1802,77 @@ inline void LM_TRANSACTION_START(const char* keyword, const char* ip, int port,
 
 /* ****************************************************************************
 *
+* LM_TRANSACTION_SET_SRV -
+*/
+inline void LM_TRANSACTION_SET_SRV(const std::string& _srv)
```

Comment:
I would really prefer to not use std::string here.
The lm-lib has gradually gone C++ but that is not at all my intention.
Please use a const char* as parameter.
Next project using lm-lib might be in C ...